git_services/git_services/init/cloner.py

```diff
@@ -90,19 +90,26 @@ def _setup_cloudstorage_symlink(self, mount_folder):
 
     @contextmanager
     def _temp_plaintext_credentials(self):
+        lfs_auth_setting = urljoin(self.git_url + "/", "info/lfs") + ".access"
```

Comment:
```suggestion
        lfs_auth_setting = urljoin(f"{self.git_url}/", "info/lfs.access")
```

Reply:
@Panaetius I had to add a few more things in the setting name but I incorporated these changes.